src/components/erc721/collectibles/collectibles_list.tsx

```diff
@@ -27,32 +36,95 @@ const MainContainer = styled.div`
 
 const FiltersMenu = styled.div`
     display: flex;
-    flex-direction: row;
+    flex-direction: column;
     margin: 0 auto 22px;
     max-width: ${themeBreakPoints.xxl};
-    padding: 24px 0 0;
+    position: relative;
     width: 100%;
+    z-index: 1;
+
+    @media (min-width: ${themeBreakPoints.md}) {
+        align-items: center;
+        flex-direction: row;
+        padding-top: 24px;
+    }
+`;
+
+const CollectiblesFilterDropdown = css`
+    margin-bottom: 25px;
+    margin-right: auto;
+    position: relative;
+
+    @media (min-width: ${themeBreakPoints.md}) {
+        margin-bottom: 0;
+        margin-right: 25px;
+    }
+
+    &:last-child {
+        margin-bottom: 0;
+
+        @media (min-width: ${themeBreakPoints.md}) {
+            margin-right: 0;
+        }
+    }
+`;
+
+const CollectiblesListSortStyled = styled(CollectiblesListSort)`
+    ${CollectiblesFilterDropdown}
+
+    z-index: 5;
+`;
+
+const CollectiblesListFilterStyled = styled(CollectiblesListFilter)`
+    ${CollectiblesFilterDropdown}
+
+    z-index: 1;
 `;
 
 const Title = styled.h1`
     color: ${props => props.theme.componentsTheme.textColorCommon};
     font-size: 18px;
     font-weight: 600;
     line-height: 1.2;
-    margin: 0;
+    margin: 0 0 25px;
+
+    @media (min-width: ${themeBreakPoints.md}) {
+        margin-bottom: 0;
+        margin-right: 30px;
+    }
 `;
 
-export const CollectiblesList = (props: Props) => {
-    const collectibles = Object.keys(props.collectibles).map(key => props.collectibles[key]);
-    return (
-        <MainContainer>
-            <FiltersMenu>
-                <Title>{props.title}</Title>
-            </FiltersMenu>
-            <CollectiblesCardList collectibles={collectibles} />
-        </MainContainer>
-    );
-};
+export class CollectiblesList extends React.Component<Props, State> {
+    public state = {
+        sortType: CollectibleSortType.NewestAdded,
+        filterType: CollectibleFilterType.ShowAll,
+    };
+
+    public render = () => {
+        const { title } = this.props;
+        const collectibles = Object.keys(this.props.collectibles).map(key => this.props.collectibles[key]);
+        const { sortType, filterType } = this.state;
+
+        return (
+            <MainContainer>
+                <FiltersMenu>
+                    <Title>{title}</Title>
+                    <CollectiblesListSortStyled currentValue={sortType} onChange={this._onChangeSortType} />
+                    <CollectiblesListFilterStyled currentValue={filterType} onChange={this._onChangeFilterType} />
```

Comment:
Basically the `CollectiblesList` holds in its state the filter & sort values, and uses `CollectiblesListSortStyled` and `CollectiblesListSfiltertyled` to display and set them via a callback.

Then, passes the values of the sort, filter and collectibles to `CollectiblesCardList` to let it render accordingly.